### ISSUE TITLE
[Bug]: Consistently logged out after a few minutes

### ISSUE THREAD

**Issue by hockeyadc** [danny-avila/LibreChat#2954]
### What happened?

I'm currently using LibreChat with an Unraid docker, installed through the community app. When I am idle for a few minutes, LibreChat logs me out. I have found the following is supposed to fix this when added to the docker-compose.override.yml:

```
services:
  api:
    command: npm run backend:dev
```

I cannot edit/access this file when it is installed through a community app in Unraid, and even if I could, changes to this file would be overridden on the next update.

### Steps to Reproduce

1. Login to your normal configuration
2. Wait 5-10 minutes
3. Try to type a prompt, it will force you to login again

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment]
You might want to consider using portainer in unraid to install LibreChat instead of the community app, this would give you more flexibility

**Comment by hockeyadc** [comment]
> You might want to consider using portainer in unraid to install LibreChat instead of the community app, this would give you more flexibility

Yeah that's an option but was hoping for a more permanent fix for the community application

**Comment by fuegovic** [comment]
> Yeah that's an option but was hoping for a more permanent fix for the community application

We did not publish the community app ourselves, maybe you could try to contact the person who made it on the unraid forum

**Comment by hockeyadc** [comment]
> > Yeah that's an option but was hoping for a more permanent fix for the community application
> 
> We did not publish the community app ourselves, maybe you could try to contact the person who made it on the unraid forum

Ok thank you, I'll give that a shot and use portainer for now
